### ISSUE TITLE
Bike data pulled in the run perspective

### ISSUE THREAD

**Issue by docsport77** [GoldenCheetah/GoldenCheetah#4458]
Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers

I have this issue with the current build as well as the snpahot build 2024-02-03T16:50:38.000Z on 2 Macs with different OSX versions.

In the run perspective which is/was preloaded I can see the colored zones displayed for some athletes correctly for running and for some others  it´s using the bike zones.

This is a run workout Zone 1, 2, 3, 2,1, we are looking at the power tab

![Bildschirmfoto 2024-03-12 um 13 12 35](https://github.com/GoldenCheetah/GoldenCheetah/assets/104032799/b6a3138c-b664-4ed7-bf15-d1dc6fbd33db)

And here a workout from another athlete with 2 times Zone 3, in this case he is using exactly the same Garmin watch as in the example above, all data is coming from Strava:

![Bildschirmfoto 2024-03-12 um 13 18 09](https://github.com/GoldenCheetah/GoldenCheetah/assets/104032799/020d04c3-2133-4c58-a272-28eb563f95f7)

I checked and in his case, these are his bike zones, his run was correct and he was not above his zones but it shows about 1.5 zones too high all the time.

Both and all others where this is happening have their run power zones setup (in the past, of course).

**Comment by docsport77** [comment]
Wow, that was fast, installed another athlete new today and added this time I did not choose to copy the layout from another athete but let load the default ones.

Still the same, all are the same and some just behaving different and the run perspective is using bike data, so what was the solution, is the run perspective updated?

**Comment by amtriathlon** [comment]
See the wiki and use the forum if you have questions or need help, the links are in you own post.
